ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -347,6 +439,63 @@
       // Process features
       this._queryFeatures = this._createFeatureIndex(data.query.features);
 
+      this.renderResults();
+    },
+
+    createPageLink : function(page, selected) {
+        var pageSpan = new Element("a", {'class':'pagenumber', 'href':'#'}).update(page);
+        if (selected) {
+           pageSpan.addClassName("selected");
+        }
+        var _this = this;
+        pageSpan.observe("click", function(ev){
+            ev.stop();
+            _this.page = ev.element().innerHTML;
+            _this.launchSearchExisting();
+        });
+        return pageSpan;
+    },
+
+    generatePaginationPageLinks : function() {
```

Comment:
I think the problem with page links is not a big deal, since having 1000+ patients is likely, 100+ page sis not, and iven if they have 1000+ pages i don't think we want to page pages :)